### DISCUSSION TITLE
obs captures virtual camera black screen?

### DISCUSSION THREAD

**Discussion by cHollower** [obsproject/obs-studio#11231] [Development Questions]
obs capture virtual camera black screen, no picture? Does it not support frames in AV_PIX_FMT_YUYV422 format?

**Comment by Fenrirthviti** [comment]
This sounds more likely to be a support request, and we are not currently accepting support requests on GitHub Discussions. Make sure you read any [pinned posts](https://github.com/obsproject/obs-studio/discussions/5206) on discussion boards before opening a new discussion. Please use our [forums](https://obsproject.com/forum/) or [Discord](https://obsproject.com/discord/) for further assistance.

Thank you!
